### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by jdcaperon**
Changes to UI. Radios to buttons and other style changes.

### CODE REVIEW

File: uqcs/templates/form.mako [hunk 1]
```diff
@@ -1,176 +1,243 @@
...
+      <p></p>
+      <p>
+        UQCS does not endorse QPAY, you should do your own research to see if
+        QPAY is right for you.
+      </p>
     </div>
+    <a role="button" class="btn btn-warning" href="https://joinuqcs.getqpay.com"
+      >Signup with QPAY</a
+    >
   </div>
```
Comment (nicklambourne) [inline uqcs/templates/form.mako:206]: needs some `<br/>` at the bottom, only really notice it when the current student part is expanded
screen shot 2019-01-23 at 12 09 50 am

File: uqcs/templates/form.mako [hunk 2]
```diff
@@ -1,176 +1,243 @@
...
+          <div class="btn-group" style="padding-top:10px">
+            <label class="btn btn-primary">
+              <input
+                name="gender"
+                type="radio"
+                value="null"
+                data-bind="checked: gender"
+                checked
+              />
+              Other / Prefer not to disclose
```
Comment (nicklambourne) [inline uqcs/templates/form.mako:79]: thoughts on putting this on the same line as Male/Female? Might wreck the symmetry though.

**Review by nicklambourne — COMMENTED** [review]
a few nitpicky suggestions, other than that looks golden

File: uqcs/templates/form.mako [hunk 3]
```diff
@@ -1,176 +1,243 @@
...
+              <input type="radio" name="year" value="2" /> 2
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="3" /> 3
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="4" /> 4
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="5" /> 5
```
Comment (nicklambourne) [inline uqcs/templates/form.mako]: should this be 5+, don't want to make @jsutton101 feel left out.

File: uqcs/templates/form.mako (hunk 2 above)
Comment (jdcaperon) [reply uqcs/templates/form.mako:79]: Yeah, this is why I left it off the same line. It also means it scales nicer to mobile devices.

File: uqcs/templates/form.mako (hunk 3 above)
Comment (jdcaperon) [reply uqcs/templates/form.mako]: Hahaha, good spot.

File: uqcs/templates/form.mako (hunk 1 above)
Comment (jdcaperon) [reply uqcs/templates/form.mako:206]: Will add.

**Pull request merged**